python/docs/dlis/investigation.rst

```diff
@@ -0,0 +1,171 @@
+Investigation hints
+===================
+
+This section might be helpful to dlisio developers or very advanced users who
+want to find a cause of a specific dlisio failure.
+
+.. note::  Unfortunately, dlisio has little debugging support. Thus failure
+           investigation at the moment is not easy.
+
+.. note::  You **might** be able to fix the file by making changes
+           just in a couple of places.
+
+.. contents:: :local:
+
+
+Prerequisites
+-------------
+
+You:
+
+ 1. Are in extreme need of reading as much data as possible from That One File
+    which dlisio fails to read, or need to investigate it
+ 2. Are very familiar with rp66 specification
+ 3. Have a good grasp on working with binary files
+ 4. Can build dlisio_ and lfp_ from source code and have enough knowledge to
+    debug it
```

Comment:
Would condense this to something like:

> The nature of a DLIS-file means that even a few incorrectly written bytes makes the file unparsable. From our experience, many files can be fixed by manual inspection, and correction of just a handful of bytes. Or by removing garbage at the end of the file. For someone with experience with working with binary files and that is familiar with the RP66 specification, it might be feasible to fix broken files manually. This section tries to give some hints to how you would go about fixing broken files